cookbooks/aws-parallelcluster-environment/test/controls/mount_shared_spec.rb

```diff
@@ -9,10 +9,10 @@
 # This file is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, express or implied.
 # See the License for the specific language governing permissions and limitations under the License.
 
-control 'mount_shared_configured' do
+control 'tag:config_mount_shared_configured' do
```

Comment:
We cannot add the tag:config because the shared folder is not attached in our CI/CD tests.

Reply:
Do you mean aws-parallelcluster-shared?

Reply:
I just double checked the code and it seems that the `/opt/parallelcluster/shared` is always exported so it should work. let's try to keep this, thanks!